server/athenian/api/experiments/preloading/entries.py

```diff
@@ -159,6 +159,20 @@ def _match_groups_to_mask(
         return (functools.reduce(operator.or_, or_masks) if or_masks
                 else np.ones(len(df), dtype=bool))
 
+    @classmethod
+    @sentry_span
+    def _find_missing_releases(cls,
+                               repos: Iterable[str],
+                               time_from: datetime,
+                               time_to: datetime,
+                               max_time_to: datetime,
+                               applied_matches: Dict[str, ReleaseMatch],
+                               spans: Dict[str, Dict[str, Tuple[datetime, datetime]]],
+                               pdb: databases.Database) -> MissingReleases:
+        missing_releases = super(PreloadedReleaseLoader, cls)._find_missing_releases(
+            repos, time_from, time_to, max_time_to, applied_matches, spans, pdb)
+        return missing_releases._replace(missing_high=[])
```

Comment:
This is nasty, tuples are supposed to be immutable + we are calling the protected method.
```suggestion
        return MissingReleases([], [], [], missing_releases.ambiguous_branches_scanned)
```

Reply:
Indeed the method name is quite misleading unfortunately: despite being called `_replace` it actually creates a new instance with that value replaced. WRT calling a protected method that's another hiccup, indeed quoting the doc:

> In addition to the methods inherited from tuples, named tuples support three additional methods and two attributes. To prevent conflicts with field names, the method and attribute names start with an underscore.

For sure not the best module of the python stdlib 😅